### DISCUSSION TITLE
Bypass Portal Apps Page?

### DISCUSSION THREAD

**Discussion by VikingDadMedic** [Budibase/budibase#3620] [Help]
Is there a way to directly have a user login to an App without having to go through the portal screen to select which app (since there is only one app)?

**Comment by hblab-hautp** [comment]
I also have same question, we have published one app but i don't want my team who is not a developer approach the portal screen
I see this question has been around for quite some time. Did you have any solution for it ? please tell me if you know !
Thank you very much !

**Reply by mjashanks** [reply]
Yes, we would like to implement this, it does get asked for a lot. I can't give you a date.

As a side note, did you know that your non-developer users see a more simple view of the app's portal: 

![image](https://user-images.githubusercontent.com/3524181/146903768-391df185-9f6f-4209-bd6e-fe167625c9fd.png)

**Reply by hblab-hautp** [reply]
yep, It's really uncomfortable.
So your team will implement it in the future, i hope it easy to migrate for my app.
Thank you !
